src/platform-integration/web/initialization.md

```diff
@@ -1,125 +1,113 @@
 ---
-title: Customizing web app initialization
+title: Flutter web app initialization
 description: Customize how Flutter apps are initialized on the web
 ---
 
-You can customize how a Flutter app is initialized on the web
-using the `_flutter.loader` JavaScript API provided by `flutter.js`.
-This API can be used to display a loading indicator in CSS,
-prevent the app from loading based on a condition,
-or wait until the user presses a button before showing the app.
+This page details the initialization process for Flutter web apps, and how this
+process can be customized.
 
-The initialization process is split into the following stages:
+## `flutter_bootstrap.js`
 
-**Loading the entrypoint script**
-: Fetches the `main.dart.js` script and initializes the service worker.
-
-**Initializing the Flutter engine**
-: Initializes Flutter's web engine by downloading required resources
-  such as assets, fonts, and CanvasKit.
-
-**Running the app**
-: Prepares the DOM for your Flutter app and runs it.
+When building your flutter app, the `flutter build web` command will produce a
+script called` flutter_bootstrap.js` in the output build directory, which
```

Comment:
```suggestion
script called` flutter_bootstrap.js` in the output build directory (`/build/web`), which
```

Reply:
The leading `/` in the suggested content is potentially problematic. I'd either add some marker for the base project directory or leave it as a relative directory, i.e. `build/web`.

Reply:
good catch!

Reply:
(fixed)